packages/admin/src/FinalForm.tsx

```diff
@@ -12,6 +14,7 @@ import { FinalFormContext, FinalFormContextProvider } from "./form/FinalFormCont
 import { SubmitError, SubmitResult } from "./form/SubmitResult";
 import { StackApiContext } from "./stack/Api";
 import { TableQueryContext } from "./table/TableQueryContext";
+const UUID = require("uuid");
```

Comment:
why do you use require here?

Reply:
Updated in this PR: https://github.com/vivid-planet/comet-admin/pull/546